homeassistant/components/verisure/__init__.py

```diff
@@ -93,16 +95,47 @@ def setup(hass, config):
     ):
         discovery.load_platform(hass, component, DOMAIN, {}, config)
 
-    def capture_smartcam(service):
+    @verify_domain_control(hass, DOMAIN)
+    async def capture_smartcam(service):
         """Capture a new picture from a smartcam."""
         device_id = service.data.get(ATTR_DEVICE_SERIAL)
```

Comment:
Please use `dict[key]` for required schema keys and keys with default schema values.

Reply:
@MartinHjelmare Can you give a bit more explain a little more, thanks

Reply:
Like this:
```suggestion
        device_id = service.data[ATTR_DEVICE_SERIAL]
```